RELEASE.md

```diff
@@ -0,0 +1,25 @@
+Release type: patch
+
+Add `GqlWsCommunicator` for testing websockets on channels.
+i.e:
+
+```python
+import pytest
+from strawberry.channels.testing import GqlWsCommunicator
+from myapp.asgi import application
+
+
+@pytest.fixture
+async def gql_communicator():
+    client = GqlWsCommunicator(application=application, path="/graphql")
```

Comment:
can we call `GqlWsCommunicator` `GraphQLWebsocketTestClient`? 

I think it might be a better name, /cc @DoctorJohn

Reply:
This fixture is user defined

Reply:
This follows channels.testing naming

Reply:
Let's make it `GraphQLWebsocketCommunicator` then. (channels provides something called `WebsocketConnector`, so I think that's matching the naming quite well).